client/landing/stepper/declarative-flow/link-in-bio-post-setup.ts

```diff
@@ -0,0 +1,43 @@
+import { LINK_IN_BIO_POST_SETUP_FLOW } from '@automattic/onboarding';
+import { useSiteSlug } from '../hooks/use-site-slug';
+import { recordSubmitStep } from './internals/analytics/record-submit-step';
+import { ProvidedDependencies } from './internals/types';
+import type { StepPath } from './internals/steps-repository';
+import type { Flow } from './internals/types';
+
+export const linkInBioPostSetup: Flow = {
+	name: LINK_IN_BIO_POST_SETUP_FLOW,
+	title: 'Link in Bio',
+	useSteps() {
+		return [ 'linkInBioPostSetup' ] as StepPath[];
+	},
+
+	useStepNavigation( currentStep, navigate ) {
+		const siteSlug = useSiteSlug();
+
+		function submit( providedDependencies: ProvidedDependencies = {} ) {
+			recordSubmitStep( providedDependencies, 'link-in-bio-post-setup', currentStep );
+
+			switch ( currentStep ) {
+				case 'linkInBioPostSetup':
+					return window.location.assign(
+						`/setup/launchpad?flow=link-in-bio&siteSlug=${ siteSlug }`
+					);
+			}
+		}
+
+		const goBack = () => {
+			return;
+		};
+
+		const goNext = () => {
+			return;
+		};
+
+		const goToStep = ( step: StepPath | `${ StepPath }?${ string }` ) => {
+			navigate( step );
+		};
```

Comment:
I imagine all of these can go?

Reply:
I don't want to pollute your PR with irrelevant changes. But making these optional would be ace. Especially with JSDoc deprecation comment discouraging people from using them and recommending `submit` [instead](https://stackoverflow.com/a/62991642/1247990).

Reply:
Hey @alshakero - Yes, maybe if possible I'd like to push this PR separately - it's already a large PR touching a lot of files. But if you'd like, I'd be happy to tackle the navigation control changes here as a quick follow up. To confirm, we'd both make them optional in the types, and effectively deprecate the methods to discourage use going forward.

Reply:
Yes yes that's what I meant! This one is perfectly shippable.